project/profiles/capacity4more/modules/c4m/og/c4m_og_invite_users/c4m_og_invite_users.module

```diff
@@ -108,3 +108,92 @@ function c4m_og_invite_users_mail_tokens(array &$replacements, array $data, $opt
   $replacements['[group:accept_invite_url]'] = $answer_link;
   $replacements['[site:name]'] = variable_get('site_name', 'Capacity 4 More');
 }
+
+/**
+ * Implements hook_form_FORM_ID_alter().
+ *
+ * Add a subject field to the invitation form.
+ */
+function c4m_og_invite_users_form__og_invite_form_alter(&$form, &$form_state, $form_id) {
+  $form['subject'] = array(
+    '#type' => 'textfield',
+    '#title' => t('Subject'),
+    '#weight' => 10,
+  );
+
+  $form['message']['#weight'] = 20;
+  $form['#submit'][0] = 'c4m_og_invite_users_form_submit';
+}
+
+/**
+ * Invitation form submit callback.
+ */
+function c4m_og_invite_users_form_submit($form, &$form_state) {
+  $group_type = $form_state['values']['group_type'];
+  $gid = $form_state['values']['gid'];
+  $message = check_plain($form_state['values']['message']);
+  $subject = check_plain($form_state['values']['subject']);
+  $invitees = explode(',', str_replace(', ', ',', $form_state['values']['invitee']));
+
+  $entity = entity_load_single($group_type, $gid);
+  $label = entity_label($group_type, $entity);
+  $field_name = key(og_get_group_audience_fields());
+  foreach ($invitees as $invitee) {
+    if (empty($invitee)) {
+      continue;
+    }
+    list($name, $uid) = explode('||', preg_replace('#^(.*)\s\[([0-9]+)\]$#', '$1||$2', $invitee));
+
+    $result = c4m_og_invite_users_invite($group_type, $gid, $field_name, $uid, $message, $subject);
+
+    if (!empty($result)) {
```

Comment:
@ybaras 
it seems that `c4m_og_invite_users_invite()` always returns __TRUE__, so this condition will always pass.